### PR TITLE
Set default country viewport after country filter change

### PR DESCRIPTION
**Pull request by kfilip94**
This small PR adds a feature which centers the viewport of the map to the selected country, works both on `Map distribution` and `Bioclimatic variables` pages:
27kbh-w1izv

### CODE REVIEW

**Comment by dfrico** [comment]
Reviewing 📝

File: src/constants.js [hunk 1]
```diff
@@ -41,3 +41,11 @@ export const COUNTRIES = [
     svgActive: tanzaniaActive
   }
 ];
+
+export const COUNTRIES_DEFAULT_VIEWPORTS = {
+  CAN: { zoom: 2, latitude: 56.1351095, longitude: -106.3460756 },
```
Comment (dfrico) [inline src/constants.js:46]: I feel like Canada's zoom could be higher. Maybe check with Ed?

File: src/pages/bioclimatic/index.js [hunk 2]
```diff
@@ -24,10 +25,13 @@ const Container = () => {
   const { country } = (match && match.params) || {};
   const currentQueryParams = useQueryParams();
   const { startYear, endYear, scenario } = currentQueryParams;
+  const [viewport, setViewport] = useState({ zoom: 4, latitude: 40, longitude: -5 });
```
Comment (dfrico) [inline src/pages/bioclimatic/index.js:28]: Does it need a default viewport? Just curiosity

**Review by dfrico — APPROVED** [review]
Looks good to me! Nice job 👍

File: src/pages/bioclimatic/index.js (hunk 2 above)
Comment (kfilip94) [reply src/pages/bioclimatic/index.js:28]: I think you're right, but I think it's still better than `null`

File: src/constants.js [hunk 3]
```diff
@@ -41,3 +41,11 @@ export const COUNTRIES = [
...
+  SWE: { zoom: 4, latitude: 62.295911, longitude: 17.861953 },
+  IDN: { zoom: 3, latitude: -4.341701, longitude: 122.389996 },
+  PER: { zoom: 4, latitude: -10.471441, longitude: -75.135125 },
+  TZA: { zoom: 5, latitude: -5.790175, longitude: 36.718777 }
+};
```
Comment (sorodrigo) [inline src/constants.js:51]: What about using bounds prop instead of viewport?

Comment (sorodrigo) [reply src/constants.js:51]: List of country bounding boxes: https://gist.github.com/graydon/11198540

Comment (dfrico) [reply src/constants.js:51]: Like here? https://github.com/Vizzuality/gfw/blob/develop/app/javascript/components/ui/map/component.jsx#L167

Comment (sorodrigo) [reply src/constants.js:51]: Like here: https://github.com/Vizzuality/gfw/blob/develop/app/javascript/components/ui/map/component.jsx#L105

Comment (kfilip94) [reply src/constants.js:51]: Good point, it seems to be a much more elegant solution, but I'd skip it for now, considering that we want to deliver working carto layers on Monday. But it should go to our tasks list for sure :+1:

Comment (dfrico) [reply src/constants.js:51]: should we merge then, and keep that in the task list?

Comment (kfilip94) [reply src/constants.js:51]: I'd vote for that

**Pull request merged**